gloperate-glkernel compile error on MSVC 2013

3>source\gloperate-glkernel\source\HemisphereDistributionKernelStage.cpp(82): error C2784: "void glkernel::sort::distance(glkernel::tkernel<V<T,P>> &,const V<T,P> &)": template-Argument für "const V<T,P> &" konnte nicht von "initializer-list" hergeleitet werden.
3>source\glkernel\include\glkernel/sort.h(18): Siehe Deklaration von 'glkernel::sort::distance'

3>source\gloperate-glkernel\source\DiscDistributionKernelStage.cpp(110): error C2784: "void glkernel::sort::distance(glkernel::tkernel<V<T,P>> &,const V<T,P> &)": template-Argument für "const V<T,P> &" konnte nicht von "initializer-list" hergeleitet werden.
3>source\glkernel\include\glkernel/sort.h(18): Siehe Deklaration von 'glkernel::sort::distance'

--- a/source/gloperate-glkernel/source/DiscDistributionKernelStage.cpp
+++ b/source/gloperate-glkernel/source/DiscDistributionKernelStage.cpp
@@ -107,7 +107,7 @@ void DiscDistributionKernelStage::regenerateKernel()
 
     std::transform(m_kernel.begin(), m_kernel.end(), m_kernel.begin(), pushCorners);
 
-    glkernel::sort::distance(m_kernel, {0.0f, 0.0f});
+    glkernel::sort::distance(m_kernel, glm::vec2(0.0f, 0.0f));
 }
 
 

--- a/source/gloperate-glkernel/source/HemisphereDistributionKernelStage.cpp
+++ b/source/gloperate-glkernel/source/HemisphereDistributionKernelStage.cpp
@@ -79,7 +79,7 @@ void HemisphereDistributionKernelStage::resizeKernel()
 void HemisphereDistributionKernelStage::regenerateKernel()
 {
     glkernel::sample::hammersley_sphere(m_kernel);
-    glkernel::sort::distance(m_kernel, {0.0f, 0.0f, 1.0f});
+    glkernel::sort::distance(m_kernel, glm::vec3(0.0f, 0.0f, 1.0f));
 }
 
 

